assets/helpers/paymentIntegrations/newPaymentFlow/visitToken/actions.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
i pretty much copied this and reducer.js from the csrf equivalent if that's right?

Reply:
maybe let's put this under `helpers` rather than `paymentIntegrations`

Reply:
done